example/src/screens/BasicPictureInPicture.tsx

```diff
@@ -118,7 +118,7 @@ export default function BasicPictureInPicture({
 
   return (
     <SafeAreaView
-      edges={['bottom', 'left', 'right']}
+      edges={Platform.isTV ? [] : ['bottom', 'left', 'right']}
```

Comment:
Nice to see they are building official TV support :)

Reply:
Actually this came from me 😅. It actually doesn't work as intended on tvOS simulator, so I opened #329 for proper fix.